src/ui/widgets/ProgressBar/progressBar.tsx

```diff
@@ -97,12 +73,40 @@ export const ProgressBarComponent = (
   }
 
   return (
-    <div className={classes.bar} style={style}>
-      <div className={classes.fill} style={fillStyle} />
-      <div className={classes.label} style={{ ...font?.css() }}>
+    <>
+      <LinearProgress
+        variant="determinate"
+        value={percent}
+        sx={{
+          position: "absolute",
+          height: height,
+          width: width,
+          border: 1,
+          borderColor: "#000000",
```

Comment:
This is an optional nitpick: the black border colour is quite noticeable, a grey that mimics the Phoebus appearance might be better.